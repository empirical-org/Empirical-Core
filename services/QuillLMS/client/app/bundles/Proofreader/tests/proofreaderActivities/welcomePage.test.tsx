import { shallow } from "enzyme";
import * as React from "react";
import WelcomePage from "../../components/proofreaderActivities/welcomePage";

describe("<WelcomePage />", () => {
  const wrapper = shallow(<WelcomePage
    onNextClick={() => {}}
  />)

  it("should render", () => {
    expect(wrapper).toMatchSnapshot();
  });

});
