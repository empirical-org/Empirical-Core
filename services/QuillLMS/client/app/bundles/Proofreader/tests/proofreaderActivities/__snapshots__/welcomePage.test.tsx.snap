// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WelcomePage /> should render 1`] = `
<div
  className="welcome-page"
>
  <div
    className="header-section"
  >
    <h1>
      Welcome to Quill Proofreader!
    </h1>
  </div>
  <div
    className="steps-section"
  >
    <div
      className="step"
    >
      <div
        className="step-number"
      >
        1
      </div>
      <h2>
        Read
      </h2>
      <p>
        Read the passage and correct the grammar and punctuation errors you find.
      </p>
      <img
        alt="Proofreader example showing a potential grammar mistake highlighted"
        src="undefined/images/illustrations/proofreader-example-proofread.svg"
      />
    </div>
    <div
      className="step"
    >
      <div
        className="step-number"
      >
        2
      </div>
      <h2>
        Review
      </h2>
      <p>
        Get feedback on each correction you made.
      </p>
      <img
        alt="Proofreader example showing a correct and incorrect revision"
        src="undefined/images/illustrations/proofreader-example-review.svg"
      />
    </div>
    <div
      className="step"
    >
      <div
        className="step-number"
      >
        3
      </div>
      <h2>
        Improve
      </h2>
      <p>
        Work on a follow-up activity that focuses on one of the errors from the passage.
      </p>
      <img
        alt="Follow-up example showing a correct revision"
        src="undefined/images/illustrations/proofreader-example-practice.svg"
      />
    </div>
  </div>
  <div
    id="button-container"
    tabIndex={-1}
  >
    <button
      className="quill-button large contained primary focus-on-light"
      onClick={[Function]}
      type="button"
    >
      Next
    </button>
  </div>
</div>
`;
